src/hb-subset-plan.cc

```diff
@@ -853,6 +854,80 @@ _normalize_axes_location (hb_face_t *face, hb_subset_plan_t *plan)
   }
   plan->all_axes_pinned = !axis_not_pinned;
 }
+
+static void
+_update_instance_metrics_map_from_cff2 (hb_subset_plan_t *plan)
+{
+  if (!plan->normalized_coords) return;
+  OT::cff2::accelerator_t cff2 (plan->source);
+  if (!cff2.is_valid ()) return;
+
+  hb_font_t *font = nullptr;
+  if (unlikely (!plan->check_success (font = _get_hb_font_with_variations (plan))))
+    return;
+
+  hb_glyph_extents_t extents = {0};
```

Comment:
This always takes 0,0 into the bounds calculation. Maybe that's okay, but it's not always correct.